javascript/javascript1/week1/age.js

```diff
@@ -0,0 +1,69 @@
+let yearOfBirth = 1995;
+let yearFuture = 2027;
+let age = yearOfBirth - yearFuture;
+console.log("You will be " + age + "years old in " + yearFuture + ".");
+
+let dogYearOfBirth = 2010;
+let dogYearFuture = 2021;
+let dogYear = dogYearFuture - dogYearOfBirth;
+const shouldShowResultInDogYears = true;
+
+if (shouldShowResultInDogYears == true) {
+  console.log("Your dog will be " + dogYear + "years old in" + dogYearFuture);
+} else {
+  console.log("Your dog will be " + age + "years old in" + dogYearFuture);
```

Comment:
`age` is my age, not the dog's age, I think you want to use another variable here.